server/models/price.py

```diff
@@ -3,29 +3,29 @@
 
 from .product import Product
 from sqlalchemy.orm import relationship
-from sqlalchemy import ForeignKey
 
 class Price(db.Model):
 
     __tablename__ = 'prices'
 
-    product = relationship('Product')
-
     primary_id = db.Column(db.Integer, primary_key=True, unique=True, autoincrement=True)
-    product_id = db.Column(ForeignKey('product.id'), nullable=False)
-    price = db.Column(db.Integer)
+    url_id = db.Column(db.String, nullable=False)
+    price = db.Column(db.Integer)    
+    currency = db.Column(db.String)
```

Comment:
I could be wrong, but is `currency` stored a symbol? Can we do USD, CAD, GBP?

Reply:
Yeah there's an issue here we need to decide on in terms of storing currency. On Amazon the domain and the currency used may not necessarily match, the user can choose a different currency. However that selection is visible at the bottom of _amazon.com_ but not others.

So the only way I found you could consistently get the right currency is just scraping it straight off of the item price, which will be formatting like: "$", "£", "€", "CDN$", etc. Right now the currency is purely cosmetic, so it shows the right currency on the UI. But for instance it doesn't differentiate between Australian dollars and US dollars, they'll both be "$".

The solution I think would be to manually create a table ourselves that matches the scraped currency symbol and the domain up to a particular 3 letter international currency acronym.

Reply:
Now, I think about, it shouldn't be **TOO** big of a deal. I was moreso just concerned on how symbols would be read on different software platforms. I'm sure SQLAlchemy already handles that issue for us so we don't need to be concerned. It's mainly because I've worked with csv files where it has been an issue previously, but I think we should be good.

Reply:
Also, in regards to the differentiating between different dollar symbols. I guess it would be more clear for the user to read "$60USD" but they would also be provided a link to go to the web page so they'll eventually realize the price.